src/browser/provider/built-in/chrome/index.js

```diff
@@ -0,0 +1,71 @@
+import browserTools from 'testcafe-browser-tools';
+import OS from 'os-family';
+import getRuntimeInfo from './runtime-info';
+import getConfig from './config';
+import { start as startLocalChrome, stop as stopLocalChrome } from './local-chrome';
+import * as cdp from './cdp';
+
+
+export default {
+    openedBrowsers: {},
+
+    isMultiBrowser: false,
+
+    async openBrowser (browserId, pageUrl, configString) {
+        var runtimeInfo = await getRuntimeInfo(configString);
+        var browserName = this.providerName.replace(':', '');
+
+        await startLocalChrome(browserName, pageUrl, runtimeInfo);
+
+        await this.waitForConnectionReady(browserId);
+
+        var cdpClientInfo = await cdp.getClientInfo(browserId, runtimeInfo);
+
+        Object.assign(runtimeInfo, cdpClientInfo);
+
+        this.openedBrowsers[browserId] = runtimeInfo;
+    },
+
+    async closeBrowser (browserId) {
+        var runtimeInfo = this.openedBrowsers[browserId];
+
+        if (cdp.canCloseBrowser(runtimeInfo))
```

Comment:
`canCloseBrowser` is unclear name.
Something like `isBrowserInEmulationMode` or `isInEmulation`.